### PR TITLE
Updated loadBundles script for connectathon and add QICore ModelInfo

### PR DESCRIPTION
**Pull request by hossenlopp**
# Summary

Latest connectathon ecqm-content-qicore-2024 bundles still have some issues with references. This adds the loadBundles script from the previous connectathon and make it fix the current issues with the bundles references, specifically with the library relatedArtifact references.

Also adds QICore ModelInfo library that is now referenced by measures.

## New behavior

`npm run loadBundles` in service project loads bundles in the `ecqm-content-qicore-2024` repo into a running measure repository.

`npm run db:setup` now inserts the QICore ModelInfo library when setting up the database.

## Code changes

New script `script/loadBundles`.

# Testing guidance

- `npm run db:reset`
- Start up MRS and front end. Ensure FHIR ModelInfo and QICore ModelInfo exist in the library list
- Have the latest `ecqm-content-qicore-2024` repo checked out adjacent to this repository.
- `npm run loadBundles`
- This script will output a lot of info about what it does. Duplicate shared libraries will report uploading with 400 errors.
- Ensure that measures appear in repository and can have $data-requirements executed on them in the UI and are able to show dependencies.

### CODE REVIEW

**Comment by p9g** [comment]
Previously reported by Elsa at https://github.com/cqframework/ecqm-content-qicore-2024/issues/47

**Comment by elsaperelli** [comment]
Refactored the code a bit so that we didn't have multiple script files with redundant code. Simply moved functions in `loadBundles.ts` to `dbSetup.ts`. Thanks Hoss for adding this!!!

File: service/scripts/dbSetup.ts [hunk 1]
```diff
@@ -10,6 +10,123 @@ dotenv.config();
...
+      for (let index = 0; index < library.relatedArtifact.length; index++) {
+        const ra = library.relatedArtifact[index];
+        // if a related artifact is using a ecqi.healthit.gov reference it needs to be
+        // changed to a madie.cms.gov reference to match the urls the resources use
+        if (
+          (ra.type === 'depends-on' || ra.type === 'composed-of') &&
+          ra.resource?.startsWith('http://ecqi.healthit.gov/ecqms/Library')
+        ) {
+          const newRef = ra.resource.replace(
+            'http://ecqi.healthit.gov/ecqms/Library',
```
Comment (hossenlopp) [inline service/scripts/dbSetup.ts:56]:
```suggestion
            'http://ecqi.healthit.gov/ecqms/Library/',
```

These need to match properly to not end up with a `//` in the new url that ends up making a broken reference for the MRS.

**Review by hossenlopp — COMMENTED** [review]
I'm unable to Request Changes since I created this PR. Theres `/` that got dropped in moving this to the dbSetup file that makes the corrected references not work.

**Review by elsaperelli — APPROVED** [review]
Lgtm!

**Pull request merged**